lib/Service/SocialApiService.php

```diff
@@ -408,9 +411,9 @@ public function updateAddressbooks(string $userId, string $offsetBook = null, st
 
 				try {
 					$r = $this->updateContact($addressBook->getURI(), $contact['UID'], $network);
-					$response = $this->registerUpdateResult($response, $contact['FN'], $r->getStatus());
+					$response = $this->registerUpdateResult($response, $contact['FN'], int($r->getStatus()));
```

Comment:
```suggestion
					$response = $this->registerUpdateResult($response, $contact['FN'], (int) $r->getStatus());
```

Reply:
ah yes, sorry I used your pipeline for testing. Didn't have a dev setup.